docs/docs/03-tools/05-context.md

```diff
@@ -0,0 +1,64 @@
+# Context
+
+GPTScript supports context provider tools which can be used to provide additional context for tool execution. Output from the context tool gets prepended to the instruction of the current tool that uses the context tool.
+
+## Writing a Context Provider Tool
+
+Here is a simple example of a context provider tool that provides additional context to search tool:
+
+```yaml
+# my-search-context-tool.gpt
+name: search_context
```

Comment:
Can we not put `name:` as it's not required.  I more care about below not using the `from` sytnax.  So the example below should just be `context: ./my-search-context-tool.gpt`

Reply:
Made the suggested change.